modules/simple-payments/simple-payments.php

```diff
@@ -40,6 +40,7 @@ private function register_scripts_and_styles() {
 
 	private function register_init_hook() {
 		add_action( 'init', array( $this, 'init_hook_action' ) );
+		add_action( 'rest_api_init', array( $this, 'register_meta_fields_in_rest_api' ) );
```

Comment:
NIT: Function name is now a bit misleading, it registers `init` and another hook.

Reply:
Do you mean `register_init_hook` is misleading or that `register_meta_fields_in_rest_api` is misleading?

I renamed it to plural `register_init_hook` → `register_init_hooks` form just now)

Both actions are called `init`s, the other one just for the rest-API.

Reply:
This is fine, no change required really.

This was more of an observation that method names like this which could be more generic become outdated as soon as we need more and different functionality.